documentation/topics/security/policies.md

```diff
@@ -54,6 +54,37 @@ There are four check types, all of which do what they sound like they do:
 
 If a single check does not explicitly authorize or forbid the whole policy, then the flow moves to the next check. For example, if an `authorize_if` check does NOT return true, this _does not mean the whole policy is forbidden_ - it means that further checking is required.
 
+### Alternative Ways To Write a Policy
+
+#### Policy with keyword list
+
+```elixir
+policies do
+  policy always(), authorize_if: always()
+end
+```
+
+#### Policy with `condition` inside `do` block
+
```

Comment:
Let's add some explanation here. Like why they might want to specify the condition this way.

Putting the condition inside the policy block can make really long condition lists easier to read.

Reply:
I have added some explanation.